src/main/java/beam/utils/gis/Network2Shapefile.java

```diff
@@ -13,7 +13,7 @@ public static void main(String[] args) {
          * [3] output network filename
          */
         Network n = NetworkUtils.createNetwork();
-        NetworkReaderMatsimV2 reader = new NetworkReaderMatsimV2(n);
+        MatsimNetworkReader reader = new MatsimNetworkReader(n);
```

Comment:
This is the new MatsimNetworkReader - which goes off the DTD - which is V2 from what I can tell...